pkg/build/gvisor.go

```diff
@@ -19,26 +21,43 @@ func (gvisor gvisor) build(params *Params) error {
 
 	config := " " + string(params.Config) + " "
 	args := []string{"build", "--verbose_failures"}
-	outBinary := ""
+	target := ""
 	if strings.Contains(config, " -cover ") {
 		args = append(args, []string{
 			"--collect_code_coverage",
 			"--instrumentation_filter=//pkg/...,-//pkg/sentry/platform/..."}...)
 	}
 	if strings.Contains(config, " -race ") {
 		args = append(args, "--features=race", "//runsc:runsc-race")
-		outBinary = "bazel-bin/runsc/linux_amd64_static_race_stripped/runsc-race"
+		target = "//runsc:runsc-race"
 	} else {
 		args = append(args, "//runsc:runsc")
-		outBinary = "bazel-bin/runsc/linux_amd64_pure_stripped/runsc"
+		target = "//runsc:runsc"
 	}
-	outBinary = filepath.Join(params.KernelDir, filepath.FromSlash(outBinary))
 	// The 1 hour timeout is quite high. But we've seen false positives with 20 mins
 	// on the first build after bazel/deps update. Also other gvisor instances running
 	// on the same machine contribute to longer build times.
 	if _, err := osutil.RunCmd(60*time.Minute, params.KernelDir, params.Compiler, args...); err != nil {
 		return err
 	}
+
+	// Find out a path to the runsc binary.
+	out, err := osutil.RunCmd(time.Minute, params.KernelDir, params.Compiler,
+		"aquery", fmt.Sprintf("mnemonic(\"GoLink\", %s)", target))
+	if err != nil {
+		return err
+	}
+
+	r, err := regexp.Compile("(?sm:.*^)\\s*Outputs: \\[(.*)\\](?sm:$.*)")
```

Comment:
What checker says, but also for a static regexp it may be handier to use a global var with MustCompile, also ensures that the regexp compiles without tests.